data/json/overmap/overmap_terrain/overmap_terrain_hardcoded.json

```diff
@@ -229,33 +229,6 @@
     "see_cost": 2,
     "flags": [ "KNOWN_DOWN", "NO_ROTATE" ]
   },
-  {
-    "type": "overmap_terrain",
```

Comment:
#49738

These needs to be migrated as you have made them rotateable.

Some examples could be found here: https://github.com/CleverRaven/Cataclysm-DDA/pulls?q=is%3Apr+migration+is%3Aclosed+author%3AZhilkinSerg+label%3A%22Map+%2F+Mapgen%22+label%3A%22Character+%2F+World+Generation%22

Reply:
Ah shoot, didn't know! let me set up a PR